zebra/zebra_rib.c

```diff
@@ -3837,7 +3837,8 @@ static void rib_meta_queue_free(struct meta_queue *mq, struct list *l,
 }
 
 static void early_route_meta_queue_free(struct meta_queue *mq, struct list *l,
-					struct zebra_vrf *zvrf)
+					struct zebra_vrf *zvrf, uint8_t proto,
```

Comment:
the zebra_vrf* can be `const`, can't it?

Reply:
sure, done